### PR TITLE
rebar3-nix: init at 0.1.0

### PR DESCRIPTION
**Pull request by dlesl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gleber** [comment]
Tag created: https://github.com/erlang-nix/rebar3_nix/tree/v0.1.0

**Review by gleber — APPROVED** [review]
I like this change!

I like this direction more than #122633

**Comment by dlesl** [comment]
Thanks, updated to use the tag

**Comment by dlesl** [comment]
@happysalada I'd like to invite you as a reviewer, not sure how though :slightly_smiling_face:

**Comment by happysalada** [comment]
@dlesl thanks! I'm not sure how I can help here, but I support the initiative!

File: pkgs/development/beam-modules/rebar3-nix/default.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{ lib, buildRebar3, fetchFromGitHub }:
+let
+  name = "rebar3_nix";
+  version = "0.1.0";
+in buildRebar3 {
+  inherit name version;
```
Comment (jonringer) [inline pkgs/development/beam-modules/rebar3-nix/default.nix:6]: feel like this is a little more readable, but don't feel strongly
```suggestion
buildRebar3 rec {
  name = "rebar3_nix";
  version = "0.1.0";
```

File: pkgs/development/beam-modules/rebar3-nix/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    owner = "erlang-nix";
+    repo = name;
+    rev = "v${version}";
+    sha256 = "17w8m4aqqgvhpx3xyc7x2qzsrd3ybzc83ay50zs1gyd1b8csh2wf";
+  };
+
+  meta = {
+    description = "nix integration for rebar3";
+    license = lib.licenses.bsd3;
+    homepage = "https://github.com/erlang-nix/rebar3_nix";
```
Comment (jonringer) [inline pkgs/development/beam-modules/rebar3-nix/default.nix:17]: Please add yourself as a maintainer when adding a new package.

https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages

File: pkgs/development/beam-modules/rebar3-nix/default.nix [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    maintainers = with lib.maintainers; [ dlesl ];
```
Comment (gleber) [inline pkgs/development/beam-modules/rebar3-nix/default.nix:16]: Please add myself as the secondary maintainer as well.

**Comment by happysalada** [comment]
yesterday there was a problem with github, many PRs had the same weird editor config failure. It's transient. When you have a moment, just force push your branch again to trigger the pipeline. The tests will probably run fine.
I can merge when all the tests pass.

**Pull request merged**